### DISCUSSION TITLE
Icon Request: icon-iron

### DISCUSSION THREAD

**Discussion by Ganginator** [FortAwesome/Font-Awesome#21211] [Icon Requests]
![iron](https://f.cloud.github.com/assets/2008667/563470/821b620e-c4da-11e2-8478-6f83a4df089d.jpeg)

An iron.
I would use it for my activities, my "irons" in the fire.

**Comment by tpeyrard** [comment]
+1

**Comment by MarGraz** [comment]
+1

**Comment by Ganginator** [comment]
Will this request ever be considered?

I've been using a generic icon I created, but would rather keep my icons in the Font Awesome wheel house.

**Comment by stefanmomm** [comment]
+1

**Comment by szczemp** [comment]
I just make this icon for hotel service.
https://u.lkn.pl/png/ZOelazko_jsBL.png
It is a modification of free fontawesome beer icon.

**Comment by jai26** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
